### PR TITLE
change: add data wrangler image uri

### PR DESCRIPTION
**Pull request by chenliu0831**
*Issue #, if available:* N/A

*Description of changes:*

Adding SageMaker Data Wrangler image URL configs.

*Testing done:* `tox`

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

#### General

- [x] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [x] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md#committing-your-change)
- [] I have passed the region in to all S3 and STS clients that I've initialized as part of this change.
- [ ] I have updated any necessary documentation, including [READMEs](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) and [API docs](https://github.com/aws/sagemaker-python-sdk/tree/master/doc) (if appropriate)

#### Tests

- [x] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have checked that my tests are not configured for a specific region or account (if appropriate)
- [ ] I have used [`unique_name_from_base`](https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/utils.py#L77) to create resource names in integ tests (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by chenliu0831** [comment]
sphinx failed for some reason, could someone help restart the build? Thanks!

**Comment by chenliu0831** [comment]
Hi, could someone take a look?

File: src/sagemaker/image_uri_config/data-wrangler.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "processing": {
+    "versions": {
+      "1.x": {
```
Comment (icywang86rui) [inline src/sagemaker/image_uri_config/data-wrangler.json:4]: Are the 1.x images guaranteed to be backward compatible?

Comment (chenliu0831) [reply src/sagemaker/image_uri_config/data-wrangler.json:4]: Correct.

**Review by ajaykarpur — APPROVED** [review]

**Review by jerrypeng7773 — APPROVED** [review]

**Review by ahsan-z-khan — DISMISSED** [review]

**Review by EthanShouhanCheng — APPROVED** [review]
LGTM

**Review by icywang86rui — APPROVED** [review]

**Pull request merged**